### PR TITLE
Converted site from CMS style to Hugo

### PR DESCRIPTION
**Pull request by rlenferink**
This pull request converts the http://community.apache.org site from the CMS style to use [Hugo](https://gohugo.io/), a static website generator.

~~Steps to do before merging:~~
~~- Check on the mailing list whether this is something comdev wants.~~
~~- Setup gitbox repository matched to this mirror (e.g. comdev-site).~~
~~- Change default branch from `trunk` to `master`.~~

Steps to do after merging:
- Setup Jenkins job to automatically trigger after new changes happen on the master branch.
- Migrate `community.apache.org` from svnpubsub to gitpubsub ([PR here](https://github.com/apache/infrastructure-puppet/pull/1753)).

An example (for demo purposes only) is available [here](http://comdev-site.roylenferink.nl/).

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+                           Apache Comdev
+              Copyright 2020 The Apache Software Foundation
+
+================================================================================
+
```
Comment (sebbASF) [inline NOTICE]: The NOTICE file should not have any extraneous text -- e.g. the === line is not required.
Also the 1st two lines should be left-justified:

http://www.apache.org/legal/src-headers.html#notice

Comment (rlenferink) [reply NOTICE]: Thanks @sebbASF! Just updated it.

Comment (rlenferink) [reply NOTICE]: The page also mentions: http://www.apache.org/legal/src-headers.html#faq-webpages

```
Does this policy also apply to content displayed on our web sites?

No. Our web sites do not have an associated NOTICE file. Instead we 
may soon be making the terms of such content explicit through a "Terms of Use" 
or "Legal Information" link in the footer of web pages. At this point, 
no action is required for Apache web sites.
```

I'll remove the NOTICE file.

Comment (sebbASF) [reply NOTICE]: Not sure that's correct.

There is a difference between the published website, and the source code that is used to create it. The public repo that creates the website may still need the NOTICE file.

Comment (rlenferink) [reply NOTICE]: Hm, should we even have a NOTICE file next to the website sources as normally the website sources are not released => `Every Apache distribution should include a NOTICE file in the top directory, along with the standard LICENSE file`. Website sources are not released as Apache distribution through ASF infrastructure as far as I know.

But of course I can be wrong about this.

If we are indeed planning to release the website sources, then a NOTICE file is definitely needed. 
I can open a LEGAL ticket to discuss whether it is needed for the current form? I was searching through LEGAL tickets but couldn't find anything.

Comment (sebbASF) [reply NOTICE]: AIUI all public ASF code repos should have N&L files - it does not matter if the code is formally released or not. A publicly accessible code repo effectively publishes the code, whether formally released or not.

Comment (rlenferink) [reply NOTICE]: Okay, re-added it. Thanks for explaining it Sebb.

**Comment by rlenferink** [comment]
@sebbASF the repository has been prepared for git (default 'master' branch & matching gitbox repo). Can you review & when good enough merge this? I'll create a Jenkins job afterwards.

**Comment by sebbASF** [comment]
Not sure which repository you want me to review, nor what the review should look at.

**Comment by rlenferink** [comment]
@sebbASF apologies for the inconvenience, with ‘this’ I meant this PR.

**Comment by sebbASF** [comment]
The NOTICE and LICENSE files look OK.
I have not checked any other parts of the PR.

**Comment by bdelacretaz** [comment]
FWIW I generated the website on my box using `hugo` with the contents of this PR and checked a few pages that I'm familiar with - it all looks good to me. 

I have also saved the current contents of http://community.apache.org/ under `archives` in this repository, so worst case we can compare and fix what's needed. So +1 to merging this PR and finishing the setup to make this live.

**Comment by rlenferink** [comment]
Thanks for verifying @bdelacretaz! Are you able to merge this? (only comdev-committers can do so)

**Comment by hhjbbarhadi** [comment]
این برنامه دانلود نمئ شود

**Pull request merged**